_posts/2015-11-15-sinatra-app.markdown

```diff
@@ -126,7 +213,7 @@ cd ..
 mkdir db
 {% endhighlight %}
 
-We are almost there. The final step in our skeleton structure for our application is to create a `config.ru` file in the root of the project. The `config.ru` file is a convention that is required by certain deployment procedures and tools (like shotgun, tux, and Heroku).
+We are almost there. The final step in our skeleton structure for our application is to create a `config.ru` file in the root of the project. The `config.ru` file is a convention that is required by certain deployment procedures and tools (like shotgun, tux, and Heroku). We also create a `Rakefile` to describe any automated tasks.
```

Comment:
we're not actually using config.ru
if we do, we can change the command to start the server to `rackup`